src/index.css

```
@@ -150,10 +150,18 @@
   /* Cards — most prominent surface, heaviest rim */
   .glass-card {
-    @apply rounded-3xl bg-white/5 backdrop-blur-3xl backdrop-saturate-150 transition-all duration-300;
-
-    border: none;
-    box-shadow: var(--glass-rim-strong), var(--shadow-glass);
+    @apply rounded-3xl border border-white/10;
+    background: linear-gradient(
+      to bottom,
+      rgba(43, 48, 58, 0.72),
+      rgba(30, 34, 42, 0.64)
+    );
```

Stylelint (17.5.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)